### PR TITLE
Created PR and issue templates

### PR DESCRIPTION
**Pull request by kapadyash**
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of creativecommons.github.io-source.

Closes #XXXXX

Added templates for creating PRs and issues on the repo. If there is another standard to be followed, do let me know!

### CODE REVIEW

File: .github/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+**Description:**
```
Comment (kgodey) [inline .github/issue_template.md]: This assumes that the issue being filed is a bug, issues can be filed for other reasons like feature requests, and usage questions.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->
+
+- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`)
+- [ ] My pull request targets the `master` branch of creativecommons.github.io-source.
```
Comment (kgodey) [inline .github/pull_request_template.md]: I would add another item to ensure that the person making the pull request has tried building the site using Lektor locally and verified that there are no build errors.

File: .github/issue_template.md (hunk 1 above)
Comment (kgodey) [reply .github/issue_template.md]: This is a good template for a bug though.

**Comment by kgodey** [comment]
Thanks for adding these, @kapadyash.

File: .github/issue_template.md (hunk 1 above)
Comment (kapadyash) [reply .github/issue_template.md]: I've made the template generic to encapsulate other reasons for an issue submission.

File: .github/pull_request_template.md (hunk 2 above)
Comment (kapadyash) [reply .github/pull_request_template.md]: Just added another bullet point to the template.

**Pull request merged**